src/utils/SpokeUtils.ts

```diff
@@ -323,14 +324,22 @@ export async function fillStatusArray(
 ): Promise<(FillStatus | undefined)[]> {
   const fillStatuses = "fillStatuses";
   const destinationChainId = await spokePool.chainId();
+
   const queries = relayData.map((relayData) => {
     const hash = getV3RelayHash(relayData, destinationChainId);
     return spokePool.interface.encodeFunctionData(fillStatuses, [hash]);
   });
-  const multicall = await spokePool.callStatic.multicall(queries, { blockTag });
-  const status = multicall.map(
-    (result: BytesLike) => spokePool.interface.decodeFunctionResult(fillStatuses, result)[0]
+
+  // Chunk the hashes into appropriate sizes to avoid death by rpc.
+  const chunkSize = 250;
```

Comment:
Is this too big?

Reply:
Currently testing w/ a 1 week lookback, but I've successfully tested chunks of 100 hashes so far, so I'm interested to find the sensible maximum that works across chains.

Reply:
Works fine w/ 1 week. I repeated w/ 2 week and instrumented `chunk()` to report on its work:

```
Produced 155 chunks of 250 from 38706 inputs.                                                  
Produced 191 chunks of 250 from 47551 inputs.                                                                                                                                                  
Produced 171 chunks of 250 from 42668 inputs.                                                  
Produced 139 chunks of 250 from 34527 inputs.                                                  
Produced 44 chunks of 250 from 10839 inputs.                                                   
Produced 42 chunks of 250 from 10322 inputs.                                                                                                                                                   
Produced 1 chunks of 250 from 104 inputs.
```

Linea is the standout w/ only 104 fills to check due to its relative age, but all other chains handled this fine. I'll leave it at 250 for now and we can tune as required in future.